src/Routers/ScheduledPushRouter.js

```diff
@@ -0,0 +1,109 @@
+import PromiseRouter from '../PromiseRouter';
+import * as middleware from '../middlewares';
+import { Parse } from 'parse/node';
+import { pushStatusHandler } from '../StatusHandler';
+
+export class ScheduledPushRouter extends PromiseRouter {
+  mountRoutes() {
+    this.route(
+      'POST',
+      '/push/sendScheduledPushes',
+      middleware.promiseEnforceMasterKeyAccess,
+      ScheduledPushRouter.handlePOST
+    );
+  }
+
+  // always returns { result: true }
+  static handlePOST(req) {
+    if (req.auth.isReadOnly) {
```

Comment:
A question for @dplewis and/or @davimacedo: why don't we need to check for the master key?  I grepped through the routers to find cases and we don't ever check, i just don't know why.

Reply:
Because of this line: https://github.com/parse-community/parse-server/pull/6079/files/5a30de782e37938d1100dfaf3f12c5e74846ff0e#diff-56b300047818a06b486cff9639ed5477R11

Reply:
riiight.  thanks.